cypress/pageObjects/signup/assertions.ts

```diff
@@ -0,0 +1,8 @@
+class SignUpPageAssertions {
+  checkSignupURL() {
```

Comment:
after merging this PR https://github.com/AzizaAlSayed/cypress-test-repo/pull/12 
you can use the checkUrlContainsValue assertion.